### PR TITLE
feat: add increment, itemGetType, SetIfNotExists API examples for docs

### PR DESCRIPTION
**Pull request by rishtigupta**
## PR Description:

Missing API list from public-dev-docs repo result:
`
SDK 'javascript' is missing 53 snippets and 0 files.
	Missing snippets: API_ConfigurationLambda,API_DictionaryFetch,API_DictionaryGetField,API_DictionaryGetFields,API_DictionaryIncrement,API_DictionaryRemoveField,API_DictionaryRemoveFields,API_DictionarySetField,API_DictionarySetFields,API_DictionaryLength,API_SetAddElement,API_SetAddElements,API_SetFetch,API_SetRemoveElement,API_SetRemoveElements,API_SetContainsElement,API_SetContainsElements,API_SetLength,API_TopicSubscribe,API_TopicPublish,API_InstantiateTopicClient,API_Increment,API_Ping,API_ItemGetType,API_KeyExists,API_KeysExist,API_SetIfNotExists,API_UpdateTtl,API_IncreaseTtl,API_DecreaseTtl,API_ItemGetTtl,API_ListFetch,API_ListConcatenateBack,API_ListConcatenateFront,API_ListLength,API_ListPopBack,API_ListPopFront,API_ListPushBack,API_ListPushFront,API_ListRemoveValue,API_ListRetain,API_SortedSetPutElement,API_SortedSetPutElements,API_SortedSetFetchByRank,API_SortedSetFetchByScore,API_SortedSetGetScore,API_SortedSetGetScores,API_SortedSetRemoveElement,API_SortedSetRemoveElements,API_SortedSetGetRank,API_SortedSetIncrementScore,API_SortedSetLength,API_SortedSetLengthByScore
`

This commit adds examples for following APIs:
- API_Increment
- API_ItemGetType
- API_SetIfNotExists

### CODE REVIEW

File: examples/nodejs/doc-examples-js-apis.ts [hunk 1]
```diff
@@ -174,6 +178,45 @@ async function example_API_Delete(cacheClient: CacheClient) {
   }
 }
 
+async function example_API_Increment(cacheClient: CacheClient) {
+  const result = await cacheClient.increment('test-cache', 'test-key', 1);
```
Comment (cprice404) [inline examples/nodejs/doc-examples-js-apis.ts:182]: might be worh doing a `set` before this, to some non-zero value, and then log the new value in your log statement?  It should be available as part of the Success result.

File: examples/nodejs/doc-examples-js-apis.ts [hunk 2]
```diff
@@ -174,6 +178,45 @@ async function example_API_Delete(cacheClient: CacheClient) {
...
+  }
+}
+
+async function example_API_SetIfNotExists(cacheClient: CacheClient) {
+  const result = await cacheClient.setIfNotExists('test-cache', 'test-key', 'test-field');
+  if (result instanceof CacheSetIfNotExists.Stored) {
+    console.log("Field 'test-field' set in key 'test-key'");
+  } else if (result instanceof CacheSetIfNotExists.NotStored) {
+    console.log(
+      "Key 'test-key' already exists in cache 'test-cache' and the field 'test-field' was not set in key 'test-key'"
```
Comment (cprice404) [inline examples/nodejs/doc-examples-js-apis.ts:211]:
```suggestion
      "Key 'test-key' already exists in cache 'test-cache', so we did not overwrite it"
```

File: examples/nodejs/doc-examples-js-apis.ts [hunk 3]
```diff
@@ -174,6 +178,45 @@ async function example_API_Delete(cacheClient: CacheClient) {
...
+    );
+  } else if (result instanceof ItemGetType.Error) {
+    throw new Error(
+      `An error occurred while attempting to set the key 'test-key' if it does not exists in cache 'test-cache': ${result.errorCode()}: ${result.toString()}`
```
Comment (cprice404) [inline examples/nodejs/doc-examples-js-apis.ts:215]:
```suggestion
      `An error occurred while attempting to call setIfNotExists for the key 'test-key' in cache 'test-cache': ${result.errorCode()}: ${result.toString()}`
```

**Review by cprice404 — COMMENTED** [review]
just a few small nits, thanks!

File: examples/nodejs/doc-examples-js-apis.ts [hunk 4]
```diff
@@ -179,9 +178,10 @@ async function example_API_Delete(cacheClient: CacheClient) {
 }
 
 async function example_API_Increment(cacheClient: CacheClient) {
+  await cacheClient.set('test-cache', 'test-key', '10');
   const result = await cacheClient.increment('test-cache', 'test-key', 1);
   if (result instanceof CacheIncrement.Success) {
-    console.log("Key 'test-key' incremented by value 1 successfully");
+    console.log(`Key 'test-key' incremented successfully. New value in key test-key: ${result.toString()}`);
```
Comment (cprice404) [inline examples/nodejs/doc-examples-js-apis.ts:184]: is there a `value` property on that success response?  or like `valueNumber`?

Comment (rishtigupta) [reply examples/nodejs/doc-examples-js-apis.ts:184]: Yes, it's there.

Comment (cprice404) [reply examples/nodejs/doc-examples-js-apis.ts]: can we call that in the log statement instead of `toString()`?

**Review by cprice404 — APPROVED** [review]
:ty:

**Pull request merged**